Update azure-security-antimalware.md

bengimblett opened the pull request:
Additional note to highlight windows server 2016 specifics after a conversation with Rakesh Narayan to confirm the behaviour of the Azure VM antimalware extension with w2016+built-in-Defender - which is to deploy policy and not SCEP. This scenario differs from the remainder of the document which goes onto describe steps for OS version < 2016.

azurecla commented:
Hi __@aashishr__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (aashishr). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

bengimblett committed:
Update azure-security-antimalware.md

Additional note to highlight windows server 2016 specifics after a conversation with Rakesh Narayan to confirm the behaviour of the Azure VM antimalware extension with w2016+built-in-Defender - which is to deploy policy and not SCEP. This scenario differs from the remainder of the document which goes onto describe steps for OS version < 2016.

GitHubber17 committed:
Update azure-security-antimalware.md
